src/css/custom.css

```diff
@@ -38,20 +38,37 @@ body {
   align-self: center;
 }
 
-/* laptops and up @media */
+/* @media laptops and up */
 @media screen
   and (min-width: 1200px) {
     .box-content {
-      max-width: 45%;
-      width: 45%;
+      max-width: 600px;
+      width: 600px;
```

Comment:
On my 1440p monitor, this is veeeery narrow. But even on a smaller screen there would be plenty of realestate to be wider. 

Also, what was wrong wth the percentage widths?